spec/install/path_spec.rb

```diff
@@ -24,7 +24,7 @@
 
       Dir.chdir(dir) do
         bundle! :install, forgotten_command_line_options(:path => "vendor/bundle")
-        expect(out).to include("installed into `./vendor/bundle`")
+        expect(out).to include("installed into `../vendor/bundle`")
```

Comment:
Why is the second dot added here, if the command running is `install --path vendor/bundle`, presumably it is installing into `$PWD/vendor/bundle`, right? And then the "installed into" would also be `./vendor/bundle`?

Reply:
It's because the `relative to cwd` behavior doesn't apply when running `bundle config`, which is what's happening in 2.0 mode, and in 1.0 mode the feature flag is disabled

Reply:
Ahh. Okay, I think that means we need to apply `relative to cwd` when running `bundle config path` as well. IMO it is identically confusing to run `bundle config path ./foo` and have that path interpreted relative to the Gemfile dir rather than cwd.

Reply:
Ok, I can look into reimplementing this behavior in the settings validator

Reply:
done !